src/components/filters/AttributeFilter/AttributeFilter.tsx

```diff
@@ -96,6 +94,38 @@ export class AttributeFilter extends React.PureComponent<IAttributeFilterProps>
         );
     }
 
+    private isInverted() {
+        const { filter } = this.props;
+
+        return filter.positiveAttributeFilter ? false : true;
+    }
+
+    private getIdentifierAndUri() {
+        const { filter } = this.props;
+        const filterType = this.isInverted() ? "negativeAttributeFilter" : "positiveAttributeFilter";
+        const identifier = filter[filterType].displayForm.identifier;
```

Comment:
This will probably not work with strict types. It needs type-guards.
```
const getIdentifier = (filter: AFM.AttribtuteFilter) => {
  if (AFM.isPositiveAttributeFilter(filter)) {
    return filter.positiveAttributeFilter.displayForm.identifier
  }
  return filter.negativeAttributeFilter.displayForm.identifier
}

const identifier = getIdentifier(filter);
```

Or more generally
```
type sharedProps = "uri" | "identifier";
const getFilterProp = (filter: AFM.AttribtuteFilter, propName: sharedProps) => {
  if (AFM.isPositiveAttributeFilter(filter)) {
    return filter.positiveAttributeFilter.displayForm[propName];
  }
  return filter.negativeAttributeFilter.displayForm[propName];
}
```

Reply:
Yep, didn't work, fixed.